source/connect/favorite-connections/import-export-ui.txt

```diff
@@ -0,0 +1,41 @@
+.. _compass-export-import-ui-landing:
+
+=========================================================
+Export and Import Favorite Connections on |compass-short|
+=========================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+To quickly connect to a shared deployment, you can export and share your
+favorite connections. Colleagues can import your favorites, or you can
+use them yourself in your other workspaces.
+
+.. important:: Export Contains Sensitive Information
+
+   By default, when you export favorite connections, database usernames
+   and passwords are included in plaintext. To protect sensitive
+   connection information, you may encrypt the exported file with
```

Comment:
[minimalism]

I think this copy was already used elsewhere, but in the interest of minimalism we could potentially remove "you may".

Also, if this `important` is included on the CLI versions of the docs as well, maybe consider single-sourcing with an include?

Reply:
I was thinking that we could potentially have both the UI and CLI pages branch off from the same landing page here. I feel like it might be redundant to have nearly the same page for each section, with the only difference really being the title. Any strong opinions/thoughts?

Reply:
Oh yeah, I like that idea!